app/atomic_gui.py

```diff
@@ -0,0 +1,26 @@
+import json
+import logging
+
+from aiohttp import web
+from aiohttp_jinja2 import template
+
+from app.service.auth_svc import for_all_public_methods, check_authorization
+from plugins.atomic.app.atomic_svc import AtomicService
+
+
+@for_all_public_methods(check_authorization)
+class AtomicGUI:
```

Comment:
This extends `BaseWorld` in skeleton so we should probably do it here.

https://github.com/mitre/skeleton/blob/68fe167bf5bc06b4f2f9cd4f95f9f796af21f1f3/app/skeleton_gui.py#L10